makemkvchapters

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
```

Comment:
I suggest (to the maintainers) to modify progressively to the «modern» shebang
`#!/usr/bin/env bash`
in `mm`.